bears/vcs/git/GitCommitBear.py

```diff
@@ -34,14 +34,21 @@ class GitCommitBear(GlobalBear):
             'issue': r'(?:\w+/\w+)?#(\S+)',
             'full issue': r'https?://gitlab\S+/issues/(\S+)',
         },
+        'bitbucket': {
+            'issue': r'(?:\w+/\w+)?#(\S+)',
```

Comment:
I think the `(?:\w+/\w+)?` here is not supported by bitbucket??

Reply:
I think bitbucket allows an issue to be referred to using 
```close #11```
or some word in between like ```close issue #11``` or ```close bug #11``` or ```close ticket #11```
So I am not sure how to go about that, I thought the above regex is for scenarios where it matches a word before issue number (possibly a link to the repo), so maybe for bitbucket we can use it to serve to support these words ?

Please correct me, I am very bad at regexe .